web/src/PickNeighbourhoodMode.svelte

```diff
@@ -262,62 +286,20 @@
   </div>
 
   <div slot="map">
-    <GeoJSON data={neighbourhoods} generateId>
+    <GeoJSON data={neighbourhoods} promoteId="name">
+      <JoinedData
+        data={hoveredNeighbourhoodFromList
+          ? [{ name: hoveredNeighbourhoodFromList, highlight: "yes" }]
+          : []}
+        idCol="name"
+      />
+
       <FillLayer
         {...layerId("neighbourhood-boundaries", false)}
         filter={["==", ["get", "kind"], "boundary"]}
         paint={{
-          "fill-color": "black",
-          "fill-opacity": hoverStateFilter(0.3, 0.5),
-        }}
-        layout={{
-          visibility: selectedPrioritization == "none" ? "visible" : "none",
-        }}
-        manageHoverState
-        bind:hovered={hoveredMapFeature}
```

Comment:
I didn't realize this till now, but it's actually a bit scary to previously have multiple FillLayers all binding to the same variable. It worked fine before because only one layer was visible at a time, but it feels confusing -- 3 simultaneous writers. I think using just one layer results in simpler code, what do you think?

Reply:
I justified 3 simultaneous writers because they were all displayed mutex, but I think what you've done here is inarguably an improvement!